pkg/machines/components/memorySelectRow.jsx

```diff
@@ -25,10 +25,11 @@ import { digitFilter, toFixedPrecision, units } from "../helpers.js";
 
 const _ = cockpit.gettext;
 
-const MemorySelectRow = ({ id, value, maxValue, initialUnit, onValueChange, onUnitChange }) => {
+const MemorySelectRow = ({ id, value, maxValue, initialUnit, onValueChange, disabled, onUnitChange }) => {
```

Comment:
You have implemented `disabled` functionality for this component, but I can't find where you use it. Have you not forgot to use that functionality somewhere?

Reply:
I don't use it anymore - thanks for noticing.